README.md

```diff
@@ -30,7 +30,7 @@ You may also optionally place the json settings directly into the `ENV_JSON` env
     username: root
     password: root
     ```
-1. As you are now logged in as `root`, there are no courses listed. Navigate to http://localhost:5001/courses/231768 or http://localhost:5001/courses/430174 to view a sample course as a superuser.
+1. As you are now logged in as `root`, there are no courses listed. 
```

Comment:
Could you add in a line about how to look up the courseid in the database and navigating to `http://localhost:5001/courses/{courseid}`?

Reply:
Absolutely. One sec